### ISSUE TITLE
Zoomable timeline for small events

### ISSUE THREAD

**Issue by richardtop** [richardtop/CalendarKit#58]
@fanstudiouk

"Another thing that I believe it's missing or would be nice to have: when we have appointments of 10-15 minutes the height is too small and you can't see the content. It would be nice to be able to zoom-in our and adjust the timeline view height."

**Comment by mukulm24** [comment]
@fanstudiouk

In TimelineView.swift
change to
var verticalDiff: CGFloat = 160
this increases the width to show more events within an hour.

**Comment by ClaudioPerroni** [comment]
Any update?

**Comment by jbarros35** [comment]
There is no verticalDiff in TimelineView.swift

**Comment by richardtop** [comment]
@jbarros35 correct. Also, it's important to not forget to call `updateStyle` method on the DayView.

**Comment by RareScrap** [comment]
We're talking about a pinch gesture, right? If so, then it is likely that I will do it in a couple of months.

**Comment by richardtop** [comment]
@RareScrap great, thanks! Looking forward to it.

**Comment by Elamuruga** [comment]
> We're talking about a pinch gesture, right? If so, then it is likely that I will do it in a couple of months.

Any update on this feature, Could you help me to implement? Thanks in advance.
